enterprise/backend/src/metabase_enterprise/serialization/v2/extract.clj

```diff
@@ -134,29 +134,23 @@
       (format "%d: %s" coll-id names))
     "[no collection]"))
 
+(defn- card-label [card-id]
+  (let [card (t2/select-one [Card :collection_id :name] :id card-id)]
+    (format "Card %d (%s from collection %s)" card-id (:name card) (collection-label (:collection_id card)))))
+
 (defn- escape-report
   "Given the analysis map from [[escape-analysis]], report the results in a human-readable format with Card titles etc."
   [{:keys [escaped-dashcards escaped-questions]}]
   (when-not (empty? escaped-dashcards)
-    (log/info "Dashboard cards outside the collection")
-    (log/info "======================================")
```

Comment:
Why did you remove all the info logging? That wasn't in the spec

Reply:
Well, the ticket says "improve logs", not "add", and the information is the same, so feels like unnecessary?..

Reply:
True